.config/gitpod/Dockerfile

```diff
@@ -67,7 +67,7 @@ FROM npm-installer as cleanup
 RUN apt-get clean && rm -rf /var/lib/apt/lists/* tmp/* 
 
 # Final stage: Setup final image and user
-FROM ubuntu:jammy
+FROM scratch
```

Comment:
Why?

Please use ubuntu or debian

Reply:
This was a suggestion to avoid including the layers from the Ubuntu:jammy image in our final image. Since we are copying the root from the last stage instead of the Ubuntu:jammy root, I was leaning towards starting from the scratch image, as proposed by @thomas. However, if you prefer to stick with the Ubuntu:jammy image, that's not a problem. In that case, I can revert it back to Ubuntu.

Reply:
@ThomasSanson please tell me what you think

Reply:
I have tested the currently generated image in single stage mode, and it has a size of 903 MB with 14 layers.

In multi-stage mode, with an inheritance of ubuntu:jammy at the end, the image size is 903 MB with 9 layers.

By testing the final stage from scratch, the image size is 902 MB with only 3 layers.

In all cases, I have performed tests with "gp validate" and also tested the installation of Nginx just in case. I have also tested by forking @AurelienToitot's repository and merging it into the master branch (mine), and running `make tests`. Everything works perfectly fine.

Now that we have this data, here are my conclusions:

- Regardless of the final image, the size varies only slightly (1 MB), so it is not necessarily significant in terms of size.
- Starting from scratch significantly reduces the number of layers, which is significant. It is even a best practice, especially since it "hides" all the previous layers:
```bash
gitpod@thomassanso-postgresqlc-sejyfqwoj6r:/workspace/postgresql_cluster$ docker history postgresql_cluster:local
IMAGE          CREATED         CREATED BY                                      SIZE      COMMENT
bcfec7f0b666   4 minutes ago   /bin/sh -c #(nop)  USER gitpod                  0B        
06ec2516807f   4 minutes ago   /bin/sh -c useradd -l -u 33333 -G sudo -md /…   9.49kB    
87e04343e6fa   4 minutes ago   /bin/sh -c #(nop) COPY dir:807d72e12e5915c80…   825MB     
5a81c4b8502e   12 days ago     /bin/sh -c #(nop)  CMD ["/bin/bash"]            0B        
<missing>      12 days ago     /bin/sh -c #(nop) ADD file:140fb5108b4a2861b…   77.8MB    
<missing>      12 days ago     /bin/sh -c #(nop)  LABEL org.opencontainers.…   0B        
<missing>      12 days ago     /bin/sh -c #(nop)  LABEL org.opencontainers.…   0B        
<missing>      12 days ago     /bin/sh -c #(nop)  ARG LAUNCHPAD_BUILD_ARCH     0B        
<missing>      12 days ago     /bin/sh -c #(nop)  ARG RELEASE                  0B     
```
vs
```bash
gitpod@thomassanso-postgresqlc-sejyfqwoj6r:/workspace/postgresql_cluster$ docker history postgresql_cluster:local
IMAGE          CREATED          CREATED BY                                      SIZE      COMMENT
de7b56120a18   12 minutes ago   /bin/sh -c #(nop)  USER gitpod                  0B        
339f6d780414   12 minutes ago   /bin/sh -c useradd -l -u 33333 -G sudo -md /…   9.49kB    
4088627bb2c6   12 minutes ago   /bin/sh -c #(nop) COPY dir:807d72e12e5915c80…   902MB 
```
This may seem a bit extreme for a simple development Dockerfile, but it fully complies with security best practices !

- Ultimately, it makes the image more stable and avoids potential sources of errors. If we need to change the environment and switch to a newer version of Ubuntu or completely change the distribution, we only need to modify one place instead of two. Therefore, starting from scratch promotes DRY (Don't Repeat Yourself) principles.

Just for information, whether it's Ubuntu, Debian, Fedora, etc., they all start from scratch, which is the base Docker image for building their operating systems. Alpine, on the other hand, starts from BusyBox.

Furthermore, since we copy the entire root to the root (`COPY --from=cleanup / /`), we overwrite everything. So, we stay within the guidelines.

With all this "expertise" (it's cool to dive deep ^^), from my side, I recommend the last layer to be "from scratch".

But honestly, it's a matter of perfectionism, so @vitabaks, as the project maintainer, I'll let you make the final decision.

Reply:
thanks for the clarification

I'm not an expert in containers (since I don't use them in production), which is why the question arose.
Let's leave it as you suggest, I entrust you with this.

@AurelienToitot @ThomasSanson thank you for your contribution, it is important for the further development of the project.